helpers/folders.ts

```diff
@@ -27,7 +27,7 @@ type FolderIdType = { zid: string | null; id: string | null };
  * Parse the given folder id and returns on object with the composing parts of the folder id
  * @param folderId
  */
-export const getFolderIdParts = (folderId: string): FolderIdType => {
+export const getFolderIdParts = (folderId?: string): FolderIdType => {
```

Comment:
This is the only parameter of this function whose purpose is to process that parameter.
So why change it to optional?
IMO if, in some context, the value that is gonna be passed as parameter can be undefined, a better approach should be to use a fallback value or to totally avoid to call this function